### DISCUSSION TITLE
A problem for a Spotify List

### DISCUSSION THREAD

**Discussion by piknahultuk** [sigma67/spotify_to_ytmusic#65] [Q&A]
I can use the script succesfull for my Spotify lists, but I have a list that contains 495 songs, when i try to use the script for the list i'm getting this error: **Could not transfer playlist Playlist Name. 'album'**, But when i try the script with list that contains 429 songs, its going successfull.

Thanks for your help.

**Comment by sigma67** [comment]
Please link the playlist
